tests/parser.cpp

```diff
@@ -80,6 +80,9 @@ static std::vector<std::string> valid_queries = {
     "contains contains 'contains'",
     "beginswith beginswith 'beginswith'",
     "endswith endswith 'endswith'",
+    "a like '*'",
```

Comment:
We should probably test that column names of `like` still work, similar to what we do above for `contains` / `beginswith` / `endswith`.